modules/ob-creating-a-s2i-build.adoc

```diff
@@ -39,7 +40,7 @@ spec:
   - name: builder-image
     value: quay.io/centos7/nodejs-12-centos7
   output:
-    image: quay.io/<repo>/s2i-nodejs-example <4>
+    image: quay.io/centos7/nodejs-12-centos7:master <4>
```

Comment:
```suggestion
```
- This needs to be kept as it. User is expected to specify the repo as per their need.